### PR TITLE
Elasticsearch cookbook v2

### PR DESCRIPTION
**Pull request by eherot**
Updates to support v2 of Elasticsearch cookbook

### CODE REVIEW

**Comment by eherot** [comment]
Relates to evertrue/et_elk#16

File: .kitchen.cloud.yml [hunk 1]
```diff
@@ -51,6 +51,7 @@ suites:
         ca_cert:
           self_signed:
             signing_key: /etc/logstash/lumberjack.key.pem
-      elasticsearch:
-        custom_config:
-          index.number_of_replicas: 0
+      et_elk:
+        elasticsearch:
+          custom_config:
+            index.number_of_replicas: 0
```
Comment (jeffbyrnes) [inline .kitchen.cloud.yml]: I’m curious; has the elasticsearch cookbook ~> 2.0 dropped these configs?

Comment (eherot) [inline .kitchen.cloud.yml]: These are needed to keep Logstash from complaining that its indexes are not replicated to at least one other Elasticsearch server. They will always be needed.

Comment (jeffbyrnes) [inline .kitchen.cloud.yml]: I was referring to the fact that the `elasticsearch` attribute namespace is now nested underneath `et_elk`, as opposed to it being top-level.

Comment (eherot) [inline .kitchen.cloud.yml]: Oh. Yes. The ES cookbook no longer uses node attributes to configure anything, so the attributes in this case are now completely local to this cookbook.

**Comment by jeffbyrnes** [comment]
After clarification above, :+1:

**Pull request merged**